src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
```

Comment:
```suggestion
export const Person = ({ person }) => {
const { name, partnerName, sex, age } = person;
```
suggest to detstructure person. it will reduce amount of code.